catalog/src/main/kotlin/com/adevinta/spark/catalog/configurator/component/Component.kt

```diff
@@ -0,0 +1,167 @@
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
+ * SOFTWARE.
+ */
+package com.adevinta.spark.catalog.configurator.component
+
+import androidx.compose.foundation.layout.Box
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.consumeWindowInsets
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.safeDrawing
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.material.icons.Icons
+import androidx.compose.material.icons.rounded.BugReport
+import androidx.compose.material.icons.rounded.Code
+import androidx.compose.material.icons.rounded.Palette
+import androidx.compose.material.icons.twotone.BugReport
+import androidx.compose.material.icons.twotone.Code
+import androidx.compose.material.icons.twotone.Palette
+import androidx.compose.runtime.Composable
+import androidx.compose.runtime.getValue
+import androidx.compose.runtime.mutableStateOf
+import androidx.compose.runtime.remember
+import androidx.compose.runtime.setValue
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.stringResource
+import androidx.compose.ui.tooling.preview.Preview
+import com.adevinta.spark.catalog.R
+import com.adevinta.spark.catalog.model.Component
+import com.adevinta.spark.catalog.model.Components
+import com.adevinta.spark.catalog.model.Configurator
+import com.adevinta.spark.catalog.util.IssueUrl
+import com.adevinta.spark.catalog.util.openUrl
+import com.adevinta.spark.components.divider.Divider
+import com.adevinta.spark.components.icons.Icon
+import com.adevinta.spark.components.icons.IconButton
+import com.adevinta.spark.components.menu.DropdownMenu
+import com.adevinta.spark.components.menu.DropdownMenuItem
+import com.adevinta.spark.components.text.Text
+import com.adevinta.spark.icons.Computer
+import com.adevinta.spark.icons.Link
+import com.adevinta.spark.icons.SparkIcons
+import com.adevinta.spark.tokens.Layout
+
+@Composable
+public fun ConfiguratorComponentScreen(
+    component: Component,
+    configurator: Configurator,
+    contentPadding: PaddingValues,
+) {
+    Column(
+        modifier = Modifier
+            .fillMaxSize()
+            .consumeWindowInsets(WindowInsets.safeDrawing)
+            .padding(contentPadding)
+            .padding(horizontal = Layout.bodyMargin),
```

Comment:
padding is applied twice?

Reply:
Nope, one is the content padding (for the top bar) and the other for the body margin